src/schema.rs

```diff
@@ -267,7 +267,8 @@ impl Name {
 #[derive(Clone, Debug, PartialEq)]
 pub struct RecordField {
     /// Name of the field.
-    pub name: String,
+//    pub name: String,
```

Comment:
I think your editor is messing up the linting, which is probably the reason why the tests are failing. Perhaps you could try running `cargo fmt`?